File: account-api/src/main/java/com/example/account/api/AccountApi.java
package com.example.account.api;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.Bean;
import org.springframework.jms.annotation.EnableJms;
import org.springframework.web.servlet.LocaleResolver;
import org.springframework.web.servlet.i18n.AcceptHeaderLocaleResolver;

import java.util.Locale;

@SpringBootApplication
@EnableJms
public class AccountApi {

    private static final Logger log = LoggerFactory.getLogger(AccountApi.class);

    public static void main(String[] args) {
        SpringApplication.run(AccountApi.class, args);
        log.info(generateLogo());
    }

    @Bean
    public LocaleResolver localeResolver() {
        AcceptHeaderLocaleResolver localeResolver = new AcceptHeaderLocaleResolver();
        localeResolver.setDefaultLocale(Locale.US);
        return localeResolver;
    }

    private static String generateLogo() {
        return  "                                                           \n" +
                "                                    _            _____ _____ \n" +
                "     /\\                            | |     /\\   |  __ \\_   _|\n" +
                "    /  \\   ___ ___ ___  _   _ _ __ | |_   /  \\  | |__) || |  \n" +
                "   / /\\ \\ / __/ __/ _ \\| | | | '_ \\| __| / /\\ \\ |  ___/ | |  \n" +
                "  / ____ \\ (_| (_| (_) | |_| | | | | |_ / ____ \\| |    _| |_ \n" +
                " /_/    \\_\\___\\___\\___/ \\__,_|_| |_|\\__/_/    \\_\\_|   |_____|\n" +
                "                                                             \n" +
                "                                                             ";
    }
}

File: services/account/account-api/src/main/kotlin/com/example/account/api/AccountApi.kt
package com.example.account.api

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.context.annotation.Bean
import org.springframework.web.servlet.LocaleResolver
import org.springframework.web.servlet.i18n.AcceptHeaderLocaleResolver
import java.util.*

@SpringBootApplication
class AccountApi {

    @Bean
    fun localeResolver(): LocaleResolver? {
        val localeResolver = AcceptHeaderLocaleResolver()
        localeResolver.defaultLocale = Locale.US
        return localeResolver
    }
}

fun main(args: Array<String>) {
    SpringApplication.run(AccountApi::class.java, *args)
}
